src/test/java/com/feelmycode/parabole/controller/CouponControllerTest.java

```diff
@@ -0,0 +1,363 @@
+        RestAssured.port = port;
+        this.spec = new RequestSpecBuilder().addFilter(
+                documentationConfiguration(this.restDocumentation))
+            .build();
+    }
+
+    @Test
+    @DisplayName("유저 본인이 보유한 쿠폰 목록 조회 : /coupon/list 에 유저 계정으로 테스트")
+    public void couponListUser() {
+
+        Long userId = 39L;
+        String userToken = this.tokenProvider.create(
+        User.builder().id(userId).email("r").password("r").nickname("r").build());    // 유저 계정
+
+        // When
+        Response resp = given(this.spec)
+            .header("authorization", "Bearer " + userToken)
+            .accept(ContentType.JSON)
+            .contentType(ContentType.JSON)
+            .filter(document("coupon-list-user",
+                    preprocessRequest(modifyUris()
+                            .scheme("https")
+                            .host("parabole.com"),
+                        prettyPrint()),
+                    preprocessResponse(prettyPrint()),
+                    responseFields(
+                        fieldWithPath("success").type(JsonFieldType.BOOLEAN).description("성공여부"),
+                        fieldWithPath("message").type(JsonFieldType.STRING).description("메세지"),
+                        fieldWithPath("data").type(JsonFieldType.OBJECT).description("응답 정보"),
+                        fieldWithPath("data.content").type(JsonFieldType.ARRAY).description("쿠폰 목록"),
+                        fieldWithPath("data.content.[].name").type(JsonFieldType.STRING)
+                            .description("쿠폰명").optional(),
+                        fieldWithPath("data.content.[].serialNo").type(JsonFieldType.STRING)
+                            .description("쿠폰 일련번호").optional(),
+                        fieldWithPath("data.content.[].sellerName").type(JsonFieldType.STRING)
+                            .description("셀러 이름").optional(),
+                        fieldWithPath("data.content.[].type").type(JsonFieldType.STRING)
+                            .description("쿠폰 유형").optional(),
+                        fieldWithPath("data.content.[].discountValue").type(JsonFieldType.NUMBER)
+                            .description("할인 값(비율/금액)").optional(),
+                        fieldWithPath("data.content.[].useState").type(JsonFieldType.STRING)
+                            .description("사용 여부").optional(),
+                        fieldWithPath("data.content.[].useDate").type(JsonFieldType.STRING)
+                            .description("사용일자").optional(),
+                        fieldWithPath("data.content.[].acquiredDate").type(JsonFieldType.STRING)
+                            .description("획득일자").optional(),
+                        fieldWithPath("data.content.[].validAt").type(JsonFieldType.STRING)
+                            .description("유효시작일자").optional(),
+                        fieldWithPath("data.content.[].expiresAt").type(JsonFieldType.STRING)
+                            .description("만료일자").optional(),
```

Comment:
일자는 전송 방식을 설명에 추가해주세요

아래는 예시입니다. 알맞게 수정해주세요.

```suggestion
                        fieldWithPath("data.content.[].useDate").type(JsonFieldType.STRING)
                            .description("사용일자 (yyyy-MM-dd'T'HH:mm:ss)").optional(),
                        fieldWithPath("data.content.[].acquiredDate").type(JsonFieldType.STRING)
                            .description("획득일자").optional(),
                        fieldWithPath("data.content.[].validAt").type(JsonFieldType.STRING)
                            .description("유효시작일자 (yyyy-MM-dd'T'HH:mm:ss)").optional(),
                        fieldWithPath("data.content.[].expiresAt").type(JsonFieldType.STRING)
                            .description("만료일자 (yyyy-MM-dd'T'HH:mm:ss)").optional(),
```